rpcapd/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+set(CMAKE_THREAD_PREFER_PTHREAD 1)
+find_package(Threads REQUIRED)
+if(NOT MSVC)
+  find_package(Threads REQUIRED)
+  if(CMAKE_USE_PTHREADS_INIT)
+    set(PCAP_LINK_LIBRARIES ${PCAP_LINK_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+  else(CMAKE_USE_PTHREADS_INIT)
+    message(WARNING "Pthreads not found. Won't be able to build rpcapd.")
+  endif(CMAKE_USE_PTHREADS_INIT)
+else(NOT MSVC)
+# tell cmake to look for pthreads-w32 so that it uses
+# our custom module in [pcap root]/cmake/FindPthreads-w32.cmake
+# instead of the standard FindThreads.cmake
+  find_package(Pthreads-w32 REQUIRED) 
+  if(PTHREADS_FOUND)
+    set(PCAP_LINK_LIBRARIES ${PCAP_LINK_LIBRARIES} ${PTHREADS_LIBRARY})
+    include_directories(${PTHREADS_INCLUDE_DIR})
+  else(PTHREADS_FOUND)
+    message(WARNING "Pthreads not found. Won't be able to build rpcapd.")
+  endif(PTHREADS_FOUND)
+endif(NOT MSVC)
+
+if(CMAKE_USE_PTHREADS_INIT OR PTHREADS_FOUND)
+
+set(BUILDING_RPCAPD 1)
+
+if(WIN32)
+set(RPCAPD_EXTRA_SOURCES win32-svc.c ${PCAP_DLL_RC})
+add_definitions(-DWIN32_LEAN_AND_MEAN -DBUILDING_RPCAPD)
```

Comment:
Where is `BUILDING_RPCAPD` used?  If it's only used to allow both the DLL and rpcapd to share a single .rc file, that's not necessary if they don't share a single .rc file.